docs/ref/models/lookups.txt

```diff
@@ -94,6 +94,38 @@ following methods:
     ``compiler.compile(expression)`` should be used. The ``compiler.compile()``
     method will take care of calling vendor-specific methods of the expression.
 
+    ``**kwargs`` may be used to supply extra context to the ``as_sql``-method
+    for that particular call. This can be useful for Expressions in
+    particular, where ``template`` can be overridden and variables intended
+    for interpolating the ``template``-string can be passed. 3rd-party
+    backends may have different SQL-syntax so they can pass these differences
+    through via ``**kwargs``.
+
+    The following example is taken from the Django ConcatPair-expression.. It
+    shows the use of ``**kwargs`` for overriding ``as_sql``-methods. Note that
+    the class do not need an ``as_sql``-method. By providing the class-variable
```

Comment:
Still not fixed. "the class do not need an" "the class does not need". Change the word "do" to "does".